src/main/java/Deadline.java

```diff
@@ -0,0 +1,10 @@
+public class Deadline extends Task {
+    private String deadline;
+    public Deadline(String description, String deadline){
+        super(description);
+        this.deadline = deadline;
+    }
+    public String toString(){
```

Comment:
introduce vertical spacing for better readability

Reply:
ok